stripe/resource_stripe_price.go

```diff
@@ -564,6 +570,8 @@ func resourceStripePriceCreate(ctx context.Context, d *schema.ResourceData, m in
 				params.Recurring.AggregateUsage = stripe.String(ToString(v))
 			case k == "usage_type" && ToString(v) != "":
```

Comment:
Same with the check for empty string.

Reply:
yes, this is where I picked the pattern from, notice this is not my addition :) But will remove this and a few other occurrences of it that were there already